eva-accession-release/src/test/java/uk/ac/ebi/eva/accession/release/configuration/batch/steps/RSAccessionsReleaseFromDBTest.java

```diff
@@ -0,0 +1,522 @@
+                "GCA_000409795.2", 60711, "CM001941.2", 100005,
+                VariantType.SNV, false, LocalDateTime.now(), 1));
+        ClusteredVariantOperationEntity cveOps_Merge_5_51 = new ClusteredVariantOperationEntity();
+        cveOps_Merge_5_51.fill(EventType.MERGED, 5L, 51L, "Original", Arrays.asList(cveInactive_5));
+
+        ClusteredVariantInactiveEntity cveInactive_51 = new ClusteredVariantInactiveEntity(new ClusteredVariantEntity(51L, "Hash-51",
+                "GCA_000409795.2", 60711, "CM001941.2", 100005,
+                VariantType.SNV, false, LocalDateTime.now(), 1));
+        ClusteredVariantOperationEntity cveOps_Merge_51_511 = new ClusteredVariantOperationEntity();
+        cveOps_Merge_51_511.fill(EventType.MERGED, 51L, 511L, "Original", Arrays.asList(cveInactive_51));
+        ClusteredVariantOperationEntity cveOps_Merge_51_512 = new ClusteredVariantOperationEntity();
+        cveOps_Merge_51_512.fill(EventType.MERGED, 51L, 512L, "Original", Arrays.asList(cveInactive_51));
+
+        ClusteredVariantInactiveEntity cveInactive_512 = new ClusteredVariantInactiveEntity(new ClusteredVariantEntity(512L, "Hash-512",
+                "GCA_000409795.2", 60711, "CM001941.2", 100005,
+                VariantType.SNV, false, LocalDateTime.now(), 1));
+        ClusteredVariantOperationEntity cveOps_Merge_512_5121 = new ClusteredVariantOperationEntity();
+        cveOps_Merge_512_5121.fill(EventType.MERGED, 512L, 5121L, "Original", Arrays.asList(cveInactive_512));
+
+        ClusteredVariantEntity cve_511 = new ClusteredVariantEntity(511L, "Hash-511",
+                "GCA_000409795.2", 60711, "CM001941.2", 100005,
+                VariantType.SNV, false, LocalDateTime.now(), 1);
+        ClusteredVariantEntity cve_5121 = new ClusteredVariantEntity(5121L, "Hash-5121",
+                "GCA_000409795.2", 60711, "CM001941.2", 100005,
+                VariantType.SNV, false, LocalDateTime.now(), 1);
+
+        cveOpsList.add(cveOps_Merge_5_51);
+        cveOpsList.add(cveOps_Merge_51_511);
+        cveOpsList.add(cveOps_Merge_51_512);
+        cveOpsList.add(cveOps_Merge_512_5121);
+        cveList.add(cve_511);
+        cveList.add(cve_5121);
+
+        // cve 6 mergeInto cve 61
+        // cve 61 is neither active nor has any further operations
+        ClusteredVariantInactiveEntity cveInactive_6 = new ClusteredVariantInactiveEntity(new ClusteredVariantEntity(6L, "Hash-6",
+                "GCA_000409795.2", 60711, "CM001941.2", 100006,
+                VariantType.SNV, false, LocalDateTime.now(), 1));
+        ClusteredVariantOperationEntity cveOps_Merge_6_61 = new ClusteredVariantOperationEntity();
+        cveOps_Merge_6_61.fill(EventType.MERGED, 6L, 61L, "Original", Arrays.asList(cveInactive_6));
+
+        cveOpsList.add(cveOps_Merge_6_61);
+
+        // cve 7 mergeInto cve 71,72
+        // cve 71 is Active
+        // cve 72 is neither active nor has any further operations
```

Comment:
It's legitimately impressive (and maybe a bit sad) reading through all these test cases to learn about the strange things that can occur in the data....